pkg/hoist/hoist_launchable.go

```diff
@@ -294,13 +305,19 @@ func (hl *Launchable) Executables(
 	// a name.
 	executableMap := make(map[string]launch.Executable)
 
-	for _, relativeEntryPoint := range hl.EntryPoints {
+	for _, relativeEntryPoint := range hl.EntryPoints.Paths {
 		absEntryPointPath := filepath.Join(hl.InstallDir(), relativeEntryPoint)
 
 		entryPointInfo, err := os.Stat(absEntryPointPath)
 		if err != nil {
-			return nil, MissingEntryPoints{
-				message: util.Errorf("missing entry point %s: %s", absEntryPointPath, err).Error(),
+			if hl.EntryPoints.Implicit {
+				// We assume the deployer knows what they're doing
+				// and there aren't supposed to be any entry points
+				return nil, nil
```

Comment:
any way `Stat` might return a different sort of error, such as no permission? If so, is it still OK and intentional that an implicit entry point will cause that error to be ignored?